pages/onboarding_group.ipynb

```diff
@@ -0,0 +1,285 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# How to onboard a new Group/Team to Operate First\n",
+    "This guide explains how you can onboard a new group to the Operate First clusters. \n",
+    "\n",
+    ":::{seealso}\n",
+    "\n",
+    "For more information about the Operate First see [Getting started with Operate First](references:operate-first-start)\n",
+    ":::\n",
+    "\n",
+    "## Prerequisites\n",
+    "- There are no prerequisites.\n",
+    "\n",
+    "\n",
+    "## Outcomes\n",
+    "\n",
+    "- [ ] A pull request against the `operate-first/apps` repository.\n",
+    "\n",
+    "The PR enables Operate First to:\n",
+    "\n",
+    "- [ ] Create a group on the Operate First fleet of clusters.\n",
+    "- [ ] Add desired users as to the created group.\n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "All manifests for all the workloads owned by Operate First Ops team are maintained in the `operate-first/apps` repository following the [Kustomize best practices](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/).\n",
+    "\n",
+    "The `cluster-scope` folder in this repo stores all privileged resources that are usually not allowed to be deployed by regular project admin and requires elevated access like cluster-admin role.\n",
+    "\n",
+    "If you want to know more about the overall design please consult Operate First's [Architectural Decision Records (ADR) archive](https://www.operate-first.cloud/blueprints/blueprint/#architectural-decisions).\n",
+    "\n",
+    "\n",
+    "## Steps"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### 1. Define important variables\n",
+    "\n",
+    "In this guide we will use a couple of facts about the project. To make it easier to follow this guide, let's define these values beforehand."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "\n",
+    "GITHUB_USERNAME = \"4n4nd\" # your github username\n",
+    "\n",
+    "GROUP_NAME=\"demo-group\"   # your desired team name\n",
+    "\n",
+    "MEMBERS_LIST = [GITHUB_USERNAME,] # list of LOWERCASE github usernames of the team members\n",
+    "\n",
+    "TEAM_MEMBERS=json.dumps([u.lower() for u in sorted(MEMBERS_LIST)]).replace(\"\\\"\", \"\\\\\\\"\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "### 2. Fork and clone the apps repository\n",
+    "\n",
+    "Please fork/clone the [operate-first/apps](https://github.com/operate-first/apps) repository. We’ll be working within this repository only.\n",
```

Comment:
one thing I think we don't take into account is if the person has already forked the repo
we can't assume their master is up to date right?
maybe we should do a git fetch upstream/master as a new branch?